AzureCalling/app/src/main/java/com/azure/samples/communication/calling/activities/CallActivity.java

```diff
@@ -413,9 +459,8 @@ private void recycleTimer() {
 
     private void setRecordingActivityNotification(final CharSequence text) {
         runOnUiThread(() -> {
```

Comment:
You'd better rename the function, for example: setActivityNotification

Reply:
Let's call this `setNoticeBannerText`

Reply:
ok, addressed.